test/suites/drift/suite_test.go

```diff
@@ -467,7 +467,7 @@ var _ = Describe("Drift", Ordered, func() {
 		env.EventuallyExpectHealthyPodCount(selector, numPods)
 		nodeClaim := env.EventuallyExpectCreatedNodeClaimCount("==", 1)[0]
 		nodeClass = env.ExpectExists(nodeClass).(*v1.EC2NodeClass)
-		expectedHash := nodeClass.Hash()
+		expectedHash := nodeClass.HashForRegion("us-west-2")
```

Comment:
These tests are not guaranteed to be running in `us-west-2`. We should dynamically discover the region and pass that in the test